### DISCUSSION TITLE
Oqtane.org website is a living example of Oqtane Framework and Oqtane Blog module?

### DISCUSSION THREAD

**Discussion by kevind2000** [oqtane/oqtane.framework#722] [General] [answered]
If yes, that's quite cool for dogfooding!

**Comment by PizzaConsole** [comment]
I believe that it is not, because it does not have a proper blog module yet

**Reply by kevind2000** [reply]
I see that Oqtane Blog Module is still under development with two issues open. Wondering what the feature gaps are? and the development/release plan?

Continue the asking, Oqtane.org is something like lightweight DNN? It performs quite well.

**Reply by PizzaConsole** [reply]
I don't have have any plans to work on the Oqtane.Blog module, but I am working on my own custom blog module

**Comment by sbwalker** [comment, marked as answer]
Oqtane.org is running DNN8. The site performs well for anonymous users because it is using caching extensively. For authenticated users the performance is painful. The plan is to migrate to Oqtane - I have a POC for a DNN to Oqtane migration tool.

**Reply by kevind2000** [reply]
Nice to know! So, there is "food": Oqtane.org and there would be "dogfooding": DNN to Oqtane Migration Tool. 

Joke aside, I suppose the migration tool mainly about Data Migration from DNN Core Entities to Oqtane Core Entities plus Content Data, like Blogs. 

Not sure the need for the tool though, at least the timing, since current Oqtane is a web application framework without many Content modules.

**Reply by sbwalker** [reply]
Oqtane uses a very similar data model to DNN - so you are correct that all of the core entities including sites, pages, modules, users, roles, permissions, settings, folders, files, etc... can all be migrated. In addition, since the most highly used module in DNN sites is the Html/Text module, the content for that module can be migrated as well. This leaves the remaining modules - some of which are "content" and others are applications which manage data.

**Reply by kevind2000** [reply]
I see. Sounds like most of Content should be migrated quickly and "DNN Custom Module" would be migrated to "Oqtane Custom Module" - Code Migration in this case, with the same application data model with its data just copied over.

Now I understand the need for DNN to Oqtane Migration Tool.

**Reply by ADefWebserver** [reply]
If version one of the migration tool only handled users accounts (including roles and permissions) and the HtmlText Module, most people would find it very useful 👍🏾
